blocks/quilted-image-block/features/quilted-image/default.jsx

```diff
@@ -0,0 +1,230 @@
+	};
+
+	// get properties from context for using translations in intl.json
+	// See document for more info https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2538275032/Lokalise+and+Theme+Blocks
+	// const { arcSite } = useFusionContext();
+	// const { locale } = getProperties(arcSite);
+	// const phrases = getTranslatedPhrases(locale);
+
+	// Account for "dark" or "light" variants y
+	/*
+  const cssClasses = [
+		BLOCK_CLASS_NAME,
+		variant === "dark" ? `${BLOCK_CLASS_NAME}--dark` : `${BLOCK_CLASS_NAME}--light`,
+	].join(" ");
+  */
+
+	return (
+		<div className={`${BLOCK_CLASS_NAME}`}>
+			<HeadingSection>
+				{headline ? <Heading>{headline}</Heading> : null}
+				<Stack>
+					<Stack direction="horizontal">
+						<div className="b-quilted-image--media-panel">
+							{/*
+              TODO : Determine if other data should be used for image alt text or if image
+              title or description needs to be brought in via searchable hooks, e.g. searchableField.
+              https://corecomponents.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/api/react-hooks.md?version=3.1
+            */}
+							<Image src={image1URL} alt={overlay1Text} />
+							<Stack className="b-quilted-image--overlay" inline={true}>
+								<Paragraph className={`b-quilted-image--overlay-text-${overlay1TextVariant}`}>
+									{overlay1Text}
+								</Paragraph>
+								{/* TODO : Change button names as per https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.00/src/components/button/index.jsx  */}
+								<Button
+									variant={getButtonVariant(button1Variant)}
+									size="small"
+									href={button1Action}
+								>
+									{button1Text}
+								</Button>
+							</Stack>
+						</div>
+						<div className="b-quilted-image--media-panel">
+							<Image src={image2URL} alt={overlay2Text} />
+							<Stack className="b-quilted-image--overlay" inline={true}>
+								<Paragraph className={`b-quilted-image--overlay-text-${overlay1TextVariant}`}>
+									{overlay2Text}
+								</Paragraph>
+								<Button
+									variant={getButtonVariant(button2Variant)}
+									size="small"
+									href={button2Action}
+								>
+									{button2Text}
+								</Button>
+							</Stack>
+						</div>
+					</Stack>
+					<div className="b-quilted-image--media-panel">
+						<Image src={image3URL} alt={overlay3Text} />
+						<Stack className="b-quilted-image--overlay" inline={true}>
+							<Paragraph className={`b-quilted-image--overlay-text-${overlay1TextVariant}`}>
+								{overlay3Text}
+							</Paragraph>
+							<Button variant={getButtonVariant(button3Variant)} size="small" href={button3Action}>
+								{button3Text}
+							</Button>
+						</Stack>
+					</div>
+				</Stack>
+			</HeadingSection>
+		</div>
+	);
+}
+
+QuiltedImage.label = "Quilted Image - Arc Block";
+
+// find matching icon in https://redirector.arcpublishing.com/pagebuilder/block-icon-library
```

Comment:
```suggestion
```

if you've found the correct block, you can remove this comment. unless you're still looking for the correct icon

Reply:
updated